app/src/main/java/com/mobiledevelopmentfoodapp/dto/Food.kt

```diff
@@ -1,10 +1,13 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Food(var name: String, var description: String, var price: Double, @SerializedName("id") var productId:Int = 0 ) {
+data class Food(
```

Comment:
Formatting and visibility modifiers. Visibility modifier best practices can be found at https://kotlinlang.org/docs/visibility-modifiers.html Formatting can be found at https://kotlinlang.org/docs/coding-conventions.html#formatting